lib/generators/social_share_button/templates/config/locales/social_share_button.pt.yml

```diff
@@ -0,0 +1,18 @@
+pt:
+  social_share_button:
+    share_to: Compartilhado por %{name}
```

Comment:
This translates to "Shared by ..." I think you meant to say "Share to ..." so that would be "Compartilhar com ..."